content/guides/embed/ui-elements/explorer.md

```diff
@@ -420,6 +420,19 @@ export default App;
 **TIP**: For a detailed flow, see [Metadata view blog post][blogpost].
 </Message>
 
+## Custom actions
+
+You can expand the actions in the **More Options** menu for files and folders in content explorer and picker. Your custom options show when user clicks the ellipsis button.
```

Comment:
Content Explorer and Content Picker

Reply:
Check out the blog for extensive description of the possible properties for filtering